recipes/django-parler/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "django-parler" %}
+{% set version = "2.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f633522c06727852978cf2f92c94a98d900dff47235412e64ed46d63e68b1f09
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - django >=1.7
+
+test:
+  requires:
+    - six
+    - django >=1.7
```

Comment:
You don't need to duplicate the django dependency here--having it above in `run:` will take care of it. Why do you need `six`?

Reply:
six is usually needed for django 1.7 but less than 2.2 (so it's more like an unspoken dependency).

Both removed as requested.